server-spi-private/src/main/java/org/keycloak/services/clientpolicy/executor/ClientPolicyExecutorProvider.java

```diff
@@ -46,4 +47,23 @@ default void executeOnEvent(ClientPolicyContext context) throws ClientPolicyExce
     String getName();
 
     String getProviderId();
+
+    @Deprecated
```

Comment:
We might not add methods for existing client registration polices. Instead of it, we could treat the re-implement client registration policies in the client policy manner., not the existing client registration manner.